hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -92,6 +94,8 @@ public static void main(final String... args) throws Exception {
         final var registry = ConstructableRegistry.getInstance();
         new SwirldsPlatformBuilder()
                 .withName("Hedera")
+                .withConfigSource(SystemEnvironmentConfigSource.getInstance())
+                .withConfigSource(SystemPropertiesConfigSource.getInstance())
```

Comment:
Allow system properties and environment variables to override other properties. Needed so various config properties can be specified in the hapi test code and take effect on the test VMs.

Reply:
Interesting.  This is a significant change in how the platform can be configured (which is currently configured _exclusively_ via two text files).

It might be wise to mention this to the platform team, devops, and release engineering so it is not a surprise to anyone.